app/Models/User.php

```diff
@@ -93,4 +93,9 @@ public function role()
     {
         return $this->belongsToMany(Role::class);
     }
+
+    public function support()
+    {
+        return $this->hasMany(Support::class);
```

Comment:
You need here to specify the column name based on the user role. and it's best to call this one `tickets` instead of `support`. (ignore this until @Aubada10 is done with roles & permissions)